api/guids/views.py

```diff
@@ -37,7 +37,10 @@ def get_serializer_class(self):
 
     def get(self, request, **kwargs):
         url = self.get_redirect_url(**kwargs)
+        view_only_token = self.request.query_params.get('view_only')
         if url:
+            if view_only_token:
+                url += '?view_only={}'.format(view_only_token)
```

Comment:
@sloria it's not possible, since `url` is the `absolute_api_v2_url` of the guid referent. Should I append all of the query params to the url or just the view only param?